org.tryton.Tryton-66.yml

```diff
@@ -0,0 +1,31 @@
+id: org.tryton.Tryton-66
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+command: tryton
+build-options:
+  share: network
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --allow=multiarch
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=xdg-desktop
+  - --filesystem=home
```

Comment:
`home` includes `xdg-desktop`. But do you really need `home` ?

Reply:
User is allowed to upload file to the tryton server. So I think it is nice to be able to select any file from home.

Reply:
if you use GtkNativeFileDialog you shouldn't even need that.

Reply:
It uses Gtk.FileChooserDialog but if I remove the option, I get the dialog but the home directory is empty.